core/job/service/job_service.go

```diff
@@ -151,8 +151,11 @@ func (j *JobService) Add(ctx context.Context, jobTenant tenant.Tenant, specs []*
 	err = j.uploadJobs(ctx, jobTenant, addedJobs, nil, nil)
 	me.Append(err)
 
-	for _, job := range addedJobs {
-		j.raiseCreateEvent(job)
+	for _, addedJob := range addedJobs {
+		j.raiseCreateEvent(addedJob)
+		if addedJob.Spec().Schedule().CatchUp() {
+			j.logger.Warn("catchup for job %s is enabled, starting from %s", addedJob.GetName(), addedJob.Spec().Schedule().StartDate().String())
```

Comment:
Do we need the start date for this? I have no strong opinion, but if it's not required for the caller, let's remove it. But if there's any particular reason, we can keep it.

Reply:
The reason is only to warn user that some runs will be created from the starting date to today. I think start date didn't give any meaning here. Will remove it